toc/rfc/rfc-0020-readiness-healthchecks.md

```diff
@@ -157,11 +157,13 @@ to route pool". When AI readiness healthcheck fails a log line is printed to AI
 logs: "Container failed the readiness health check. Container marked not ready
 and removed from route pool".
 
-#### App events
+#### App Audit Events
 
-When AI readiness healthcheck succeeds a new application event is emitted:
-"app.ready". When AI readiness healthcheck fails a new event is emitted:
-"app.notready".
+When the liveness healthchecks fail, it results in the following audit events:
+`audit.app.process.crash` and `audit.app.process.rescheduling`.
```

Comment:
audit.app.process.rescheduling is sent when an app is terminated because of cell evacuation. I don't think that it is related to the liveness healthcheck at all.

[ccng #2327](https://github.com/cloudfoundry/cloud_controller_ng/issues/2327)
https://docs.cloudfoundry.org/running/managing-cf/audit-events.html#when-are-events-created

Reply:
Updated the wording. How does it look now?